CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# This file provides an overview of code owners in the "asyncapi-react" repository.
+
+# Each line is a file pattern followed by one or more owners.
+# The last matching pattern has the most precedence.
+# For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
+
+# These are the default owners for the whole content of this repository. The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
+* @ivangsa @derberg @asyncapi-bot-eve
```

Comment:
@ivangsa to have all stuff work as expected we need this. Are you ok with this? to be a member of AsyncAPI org and be here as codeowner. I added myself just to be here to help with review and approval